app/src/main/res/layout/submitted_answer_item.xml

```diff
@@ -52,6 +56,7 @@
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:background="@drawable/submitted_answer_background"
+      android:contentDescription="@{accessibleAnswer ?? submittedAnswer}"
```

Comment:
How does ``??`` work? Is it a null check, or something else? I'd expect accessibleAnswer to be empty when it's not defined elsewhere.

Did you test this with other interactions to make sure the current accessibility behavior is being retained?

Reply:
is it a null check, yes and I did check it with other interactions their behavior are intact.

Reply:
Ack, thanks for clarifying.